guides/shared/files/vat-cat-line.adoc

```diff
@@ -3,7 +3,7 @@
 
 VAT information on line level is provided in the class `cac:ClassifiedTaxCategory`.
 
-If the class is used, each line shall have the item VAT category code, and for all VAT categories except "Not subject to VAT" (Category code O), the VAT rate shall be provided.
+Each line shall have the item VAT information including category code and rate. A VAT rate of zero is applied for items "Not subject to VAT" (Category E, Z, G, O, K).
```

Comment:
The essense of the text is that the lines always shall have VAT category code except when the VAT category is O. I suggest that text is 

Each line shall have the item VAT information including category code and rate except when the invoice is "Not subject to VAT". (VAT category code on document level =O).

Reply:
Each line shall have the item VAT information including category code and percentage rate except when the invoice is "Not subject to VAT" (VAT category code on document level =O) then percentage shall not be given on line level.